pages/about.html

```diff
@@ -4,9 +4,79 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Yun Liu Intro</title>
+  <link rel="stylesheet" href="/style/style.css">
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <div id="logo"><a href="/pages/index.html" rel="about" class="logo">Home</a></div>
+      <ul>
+        <li><a href="/pages/portfolio.html" rel="portfolio">My Portfolio</a></li>
+        <li><a href="/pages/about.html" rel="about">About Me</a></li>
+        <li><a href="#social-footer" rel="contact">Contact Me</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="hero">
+      <div class="hero-text">
+        <h1>Hello again!
+          <br>Greetings from Yun Liu</h1>
+      </div>
+    </section>
+    <section class="content">
+      <div class="skills">
+        <h2>Coding Skills</h2>
+        <p> I am Bootcamp trained & self learning Software Developer. Ex-Mortgage & Credit Financing professional.
+        </p>
+        <p>During my time at Ada, I spent 6 month full-time on campus and received rigorous coursework. I will also participate in a 5 month internship at a midsize to MAANG company that provides industry-level training and hands on experience in the following:
+        </p>
+        <ul>
+          <li>Fundamentals of Computer Science</li>
+          <li>Python</li>
+          <li>Object Oriented Programming</li>
+          <li>SQL, Flask, SQLAlchemy, RESTful APIs</li>
+          <li>Javascript, React, HTML, CSS</li>
+          <li>Command Line, Collaborating with Git</li>
+          <li>Test Driven Development</li>
+          <li>Agile methodologies; Scrum Sprint Cycle</li>
+        </ul>
+      </div>
+      <div class="P&E">
+        <h2>Projects & Experience</h2>
+        <p>I have done multiple projects in both backend and frontend and built APIs to properly handle HTTP requests.
+        </p>
+        <p>Please visit my portfolio page located in the navigation bar for more detail.
+        </p>
+      </div>
+      <div class="more">
+        <h2>Outside of Coding</h2>
+        <p>Outside of coding, I am also an active Amazon FBA seller and an Airbnb Superhost.
+        </p>
+        <p>It's my inspiration to learn how the backend of all these platforms work for users and businesses. And I love to travel around the world and explore new topics. I am excited to combine my curiosity about the world and coding skills to continue exploring.
+        </p>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div id="social-footer">
+      <p>&copy; Copyright: 2022, Yun Liu All Rights Reserved</p>
+      <ul>
+        <li>
+          <img src="https://img.icons8.com/color/48/null/gmail--v2.png" alt=" "/>
```

Comment:
The alt text should not be an empty string. People who are blind or have low vision depend on screen readers to tell them what is on a webpage. Screen readers use the alt text to explain to the user what the image is. This tricks the HTML validator into not throwing an error, but it is still not good for the user who will not understand what these images are.